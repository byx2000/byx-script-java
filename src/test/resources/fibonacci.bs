var fib1 = n => {
    if (n == 1 || n == 2) {
        return 1
    }
    return fib1(n - 1) + fib1(n - 2)
}

function fib2(n) {
    if (n == 1 || n == 2) {
        return 1
    }
    return fib2(n - 1) + fib2(n - 2)
}

Console.println(fib1(10))
Console.println(fib1(20))
Console.println(fib2(10))
Console.println(fib2(20))
