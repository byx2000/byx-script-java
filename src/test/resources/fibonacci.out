55
6765
55
6765
